src/ensembl/src/shared/accordion/AccordionContainer/AccordionContainer.tsx

```diff
@@ -0,0 +1,165 @@
+// tslint:disable:max-classes-per-file
```

Comment:
do we need this?

Reply:
We need to include this line as the linter is not happy with having two class defined in a single file. We could alternatively disable this rule in the tslint config if that's okay?

Reply:
I think disabling the rule should be okay; I can't think of a reason why we need the arbitrary limitation of no more than 1 class per file. @imran-ebi ?

Reply:
Yes we can do that. Anyway, I doubt we will be having more than one class when we switch most of these components to functional components.